primitives/src/script/mod.rs

```diff
@@ -47,9 +47,15 @@ pub type ScriptSig = Script<ScriptSigTag>;
 /// A script public key (scriptPubKey).
```

Comment:
I pulled @jamillambert up for this exact style of docs in relation to a scriptPubkey and here I'm doing it myself. Is it ok to refer to this concept like this (seems to me that public keys can occur in many places in a script and this concept is not referring to that so should be a single word)?

Reply:
We have it already also here:

```rust
/// A script public key (scriptPubKey).
pub type ScriptPubKeyBuf = ScriptBuf<ScriptPubKeyTag>;
```

Git blame says you wrote that @apoelstra so I'm guessing this style is ok? Anyways ...